Backend/src/app/controllers/authControllers.ts

```diff
@@ -69,5 +70,12 @@ export class Auth_Controllers
             success: true,
             message: "Logged out"
         })
-    }) 
+    })
+
+    public authorizeRoles = (...roles: any[]) =>  {
```

Comment:
I don't think it is a good practice to use any, weird eslint/ts didnt catch this.

```suggestion
    public authorizeRoles = (...roles: Role[]) =>  {
```

And create a Role enum:
```js
    enum Role {
        Admin = 'admin',
        ...
    }
```